README.md

```diff
@@ -20,3 +20,7 @@ There is currently only one implementation in TypeScript:
 ## License
 
 SPDX-License-Identifier: [CC-BY-SA-4.0](LICENSE)
+
+## Code of Conduct
+
+Trust, partnership, simplicity and passion are our core values we live and breathe in our daily work life and within our projects. Our open-source projects are no exception. We have an active community which spans the globe and we welcome and encourage participation and contributions to our projects by everyone. We work to foster a positive, open, inclusive and supportive environment and trust that our community respects the micro:bit code of conduct. Please see our [code of conduct](https://microbit.org/safeguarding/) which outlines our expectations for all those that participate in our community and details on how to report any concerns and what would happen should breaches occur.
```

Comment:
This README keeps 80 characters per line, so let's break this paragraph in that line length (also separated the las sentence in a new paragraph so that it's not all in a big chunk).

```suggestion
Trust, partnership, simplicity and passion are our core values we live and
breathe in our daily work life and within our projects. Our open-source
projects are no exception. We have an active community which spans the globe
and we welcome and encourage participation and contributions to our projects by
everyone. We work to foster a positive, open, inclusive and supportive
environment and trust that our community respects the micro:bit code of conduct.

Please see our [code of conduct](https://microbit.org/safeguarding/) which
outlines our expectations for all those that participate in our community and
details on how to report any concerns and what would happen should breaches
occur.
```

Reply:
Didn't know about the 80 characters, LGTM